### PR TITLE
[Doc][Improve]add transform v2 doc & remove transform v1 doc

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

File: docs/en/transform-v2/replace.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+## Options
+
+| name           | type   | required | default value |
+| -------------- | ------ | -------- |---------------|
+| replace_field  | string | yes       |               |
+| pattern        | string | yes      | -             |
+| replacement    | string | yes      | -             |
+| is_regex       | boolean| no       | false         |
+| replace_first  | boolean| no       | false         |
```
Comment (ashulin) [inline docs/en/transform-v2/replace.md]:
```suggestion
| name           | type   | required | default value |
| -------------- | ------ | -------- |---------------|
| replace_field  | string | yes      |               |
| pattern        | string | yes      | -             |
| replacement    | string | yes      | -             |
| is_regex       | boolean| no       | false         |
| replace_first  | boolean| no       | false         |
```

**Review by hailin0 — DISMISSED** [review]

File: docs/en/transform-v2/replace.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    replacement = "b"
+    is_regex = true
+  }
+}
+
+sink {
+  Console {
+    source_table_name = "fake1"
+  }
+  Assert {
```
Comment (hailin0) [inline docs/en/transform-v2/replace.md]: remove Assert sink

File: docs/en/transform-v2/filter-rowkind.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    result_table_name = "fake1"
+    exclude_kinds = ["INSERT"]
+  }
+}
+
+sink {
+  Console {
+    source_table_name = "fake1"
+  }
+  Assert {
```
Comment (hailin0) [inline docs/en/transform-v2/filter-rowkind.md]: remove Assert sink

File: docs/en/transform-v2/replace.md (hunk 2 above)
Comment (EricJoy2048) [reply docs/en/transform-v2/replace.md]: done

File: docs/en/transform-v2/filter-rowkind.md (hunk 3 above)
Comment (EricJoy2048) [reply docs/en/transform-v2/filter-rowkind.md]: done

**Review by hailin0 — APPROVED** [review]
LGTM

**Review by ic4y — APPROVED** [review]
+1

**Pull request merged**